omnibus/config/software/elfutils.rb

```diff
@@ -44,9 +44,13 @@
       "--disable-nls",
       "--enable-pic"
     ]
-  
+
     configure(*configure_options, env: env)
-  
+
     make "-j #{workers}", env: env
     make "install", env: env
+
+    delete "#{install_dir}/embedded/lib/libdw.a"
```

Comment:
I think there is a `BUILD_STATIC` option for elfutils https://github.com/roolebo/elfutils/blob/cff53f1784c9a4344604bedf41b7d499b3eb30d5/configure.ac#L326

Reply:
Indeed, but AFAICS it's tied to `--enable-gcov` / `--enable-gprof`
I don't really understand why it would build the static library given that we don't provide any of those arguments and that they default to `no` but I'll look into it

Reply:
Oh OK if I understand correctly `BUILD_STATIC` is used to force their binaries to statically link with their libraries, which is consistant with gprof/gcov scenario.

However, there is no way to disable static libraries building, they will always build both static and shared libraries as they don't seem to rely on libtool, but rather extend each `install` rule to build & install both flavors of the libraries

AFAICS we can't tweak the behavior without patching their build system